exercises/reverse-string/canonical-data.json

```diff
@@ -12,25 +12,25 @@
       "expected": ""
     },
     {
-      "description": "reverse a string",
+      "description": "a word",
       "property": "reverse",
       "input": "robot",
       "expected": "tobor"
     },
     {
-      "description": "reverse a string with a capital letter",
+      "description": "a capitalized word",
       "property": "reverse",
       "input": "Ramen",
       "expected": "nemaR"
     },
     {
-      "description": "reverse a string with punctuation",
-      "property": "reverse",
+      "description": "a sentence with punctuation",
+      "property": "reverse",  
       "input": "I'm hungry!",
       "expected": "!yrgnuh m'I"
     },
     {
-      "description": "reverse a string and check if it is the same as input string (Palindrome)",
+      "description": "a string that reads the same backward as forward (i.e. palindrome)",
```

Comment:
Why are you defining palindrome here?

Reply:
I don't think everybody knows what a palindrome is, and personally, I'd rather have it defined here than have to google it. If you feel strongly we don't need to define it, please state why you think everybody knows this. Perhaps wording can change some.